demo-simple/src/main/res/layout/merge_download_batch_status.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:parentTag="android.support.constraint.ConstraintLayout">
+
+  <TextView
+    android:id="@+id/download_batch_status"
+    android:layout_width="match_parent"
+    android:layout_height="0dp"
+    android:layout_marginEnd="8dp"
+    android:layout_marginLeft="8dp"
+    android:layout_marginRight="8dp"
+    android:layout_marginStart="8dp"
+    android:layout_marginTop="8dp"
```

Comment:
These margins are always the same... worth extracting these (8 and 16) to res? 
Not a blocking eh, it's good already for the demo